packages/flutter_goldens/lib/flutter_goldens.dart

```diff
@@ -63,9 +85,12 @@ class FlutterGoldenFileComparator implements GoldenFileComparator {
   }) async {
     defaultComparator ??= goldenFileComparator;
 
-    // Prepare the goldens repo.
+    const Platform platform = LocalPlatform();
     goldens ??= GoldensClient();
```

Comment:
Wy instantiate this if we never call prepare on it? (Maybe move this into the if)?